### DISCUSSION TITLE
Obsidian Plugin 추천

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#87] [General]
# Obsidian Plugin 추천

옵시디언 플러그인 팔러 왔습니다. 혹시 옵시디언을 사용하시는 분들은 유용하게 사용하실 수 있는 플러그인들을 엄선해서 글을 작성했으니, 유용한 플러그인 얻어가시길 바랍니다. 옵시디언을 아직 안 써보신 분들은 이렇게 다양한 플러그인들이 있으니... 옵시디언에 관심 주시면 감사하겠습니다 ㅋㅅㅋ 간단하게 커뮤니티 플러그인 사용하는 법 설정 - 커뮤니티 플러그인으…

https://vero.wiki/obsidian-plugin/

**Comment by dmstjdhdh** [comment]
내가 작성한 노트의 흐름을 확인하거나, 팀원들과 회의내용을 기록하고 리뷰할 때 MindMap 플러그인도 좋은것 같아요.

이미 작성된 마크다운 문서를 기반으로 자동완성되는거라, 사용자가 자유롭게 맵을 변경하는데에는 제한적이지만, 단순하게 쓴 글을 확인하는 목적으로는 추천드립니다.
